src/app/(auth)/login/Login.jsx

```diff
@@ -0,0 +1,101 @@
+"use client";
+import React, { useState } from "react";
+import Link from "next/link";
+import RegisterModal from "./RegisterModal";
+import { supabase } from "../supabase";
+
+export default function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [showRegisterModal, setShowRegisterModal] = useState(false);
+
+  const handleEmailChange = (e) => {
+    setEmail(e.target.value);
+  };
+
+  const handlePasswordChange = (e) => {
+    setPassword(e.target.value);
+  };
+
+  const handleSignIn = async (e) => {
+    e.preventDefault();
+    if (!validateEmail(email)) {
+      alert("Please enter a valid email address.");
+      return;
+    }
+    // if (!validatePassword(password)) {
+    //   alert("Please enter a valid password. Password must be at least 8 characters long.");
+    //   return;
+    // }
+    const { data, error } = await supabase.auth.signInWithPassword({
+      email,
+      password,
+    });
+
+    if (error) {
+      alert(error.message);
+    } else {
+      console.log(data);
+      window.location.href = '/';
```

Comment:
If this is a re-direct after login success, you would want to use Link form next/link or useRouter from next/navigation. It looks like Link is already imported